tests/llms/llm_utils.sh

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+function prepare_models() {
+
+    if [ $# -lt 2 ]; then
+        echo "Usage: prepare_models <directory> <model1> [model2] ..."
+        return
+    fi
+
+    local model_path=$1
+    shift
+    mkdir -p ${model_path}
+    python3 -m pip install huggingface_hub[cli] --user
+    # Workaround for huggingface-cli reporting error when set --cache-dir to same as default
+    local extra_args=""
+    local default_model_dir=$(readlink -m ~/.cache/huggingface/hub)
+    local real_model_dir=$(echo ${model_path/#\~/$HOME} | xargs readlink -m )
```

Comment:
What `${model_path/#\~/$HOME}` is supposed to do?

I would assume it's some non-POSIX, Bash specific shell syntax, but this script is missing an interpreter hashbang...

In POSIX syntax, `${model_path#~/}` would make it relative to home dir, same as`${model_path#$HOME/}`, if $HOME is defined.

(Also, `function` is redundant for function declarations, and POSIX compliance in general can be checked  on Debian/Ubuntu with `devscripts` package provided `checkbashishms` tool.)

Reply:
> And yes, it's bash specific.

Ok, please add Bash hashbang for the script.

> Because the model_path is set outside of these test scripts, and sometimes contains ~, we need to do this workaround due the the huggingface-cli issue.

If something is giving variables with unexpanded `~` characters to a script, that's a bug to be fixed, not something to work around!